src/AdminAcceptanceTests.Actions/Pages/OrganisationDashboard.cs

```diff
@@ -24,5 +25,15 @@ public bool LinksToManageOrganisationsAreDisplayed()
         {
             return Driver.FindElements(Pages.OrganisationDashboard.OrganisationLinks).Count > 0;
         }
+
+        public string SelectOrganisation()
```

Comment:
how about passing an index, where the default is null. if default is null, randommm